funding/commitment_type_negotiation.go

```diff
@@ -24,19 +24,20 @@ var (
 // negotiateCommitmentType negotiates the commitment type of a newly opened
 // channel. If a channelType is provided, explicit negotiation for said type
 // will be attempted if the set of both local and remote features support it.
-// Otherwise, implicit negotiation will be attempted.
+// Otherwise, implicit negotiation will be attempted. A bool is returned
+// indicating whether a zero-conf channel was negotiated.
 func negotiateCommitmentType(channelType *lnwire.ChannelType,
 	local, remote *lnwire.FeatureVector,
-	mustBeExplicit bool) (bool, lnwallet.CommitmentType, error) {
+	mustBeExplicit bool) (bool, lnwallet.CommitmentType, bool, error) {
```

Comment:
Why do we need to indicate zero conf or not if we have a channel type feature bit that can be checked?

Reply:
its not necessary good catch, can just check for non-nil channel type and for zeroconf bit